htaccess

```diff
@@ -0,0 +1,45 @@
+# This htaccess file is used to do language redirects based on the first
+# language in the Accept-Language HTTP header.
+#
+# Fallback order: lang-region -> lang -> en
+
+RewriteEngine On
+
+# Since the redirects in this file vary based on the Accept-Language header, we
+# set the Vary header so that reverse proxy caches (e.g. Cloudflare) know how to
+# key the cache.
+<IfModule mod_headers.c>
+  Header set Vary "Accept-Language"
+</IfModule>
+
+# Record the base URI in ENV::BASE.
+# https://stackoverflow.com/questions/21062290/set-rewritebase-to-the-current-folder-path-dynamically
+RewriteCond %{REQUEST_URI}::$1 ^(.*?/)(.*)::\2$
+RewriteRule ^(.*)$ - [E=BASE:%1]
+
+# First, append Accept-Language to the URL.
+RewriteRule ^$ %{REQUEST_URI}%{HTTP:Accept-Language} [L,R=301]
+
+# Then, hackily extract the first preferrered language by stripping everything
+# from a comma or semicolon onward.
+RewriteCond %{REQUEST_URI} [,;]
+RewriteRule ^([^,;]+).*$ %{ENV:BASE}$1 [L,R=301]
+
+# Rewrite any HTML/JS/CSS/PDF legacy links from http://mixxx.org/manual/2.1/* to
+# http://mixxx.org/manual/2.1/en/*
+RewriteCond $1 \.(html|js|css|pdf|gif|png|jpg)$
```

Comment:
+svg ?

Reply:
I think there were no SVGs in the manual we've published up til now.  Though I guess there are no jpgs either.

Reply:
Done